cni/network/multitenancy_test.go

```diff
@@ -47,7 +57,12 @@ func (c *MockCNSClient) RequestIPAddress(_ context.Context, ipconfig cns.IPConfi
 	return c.request.result, c.request.err
 }
 
-func (c *MockCNSClient) ReleaseIPAddress(_ context.Context, ipconfig cns.IPConfigRequest) error {
+func (c *MockCNSClient) RequestIPs(_ context.Context, ipconfig cns.IPConfigRequest) (*cns.IPConfigsResponse, error) {
+	c.require.Exactly(c.requestIPs.ipconfigArgument, ipconfig)
+	return c.requestIPs.result, c.requestIPs.err
+}
+
+func (c *MockCNSClient) ReleaseIPs(_ context.Context, ipconfig cns.IPConfigRequest) error {
```

Comment:
By 2 functions with the same name do you mean the RequestIPs and RequestIPAddress? Right now I left it so that the RequestIPAddress tests the old API (using IPConfigResponse) while RequestIPs is supposed to test the new one (using IPConfigsResponse). When we take out the old API RequestIPAddress can be removed with it.